README.md

```diff
@@ -1 +1,27 @@
-# bitnaughts.unity
+![image](images/banner.png)
+
+# BitNaughts Unity Project
+
+Interested in setting up the BitNaughts development environment? You're at the right place!
+
+## Prerequisites
+
+- [Unity 2018.3.0f](https://unity3d.com/get-unity/download/archive)
+- [Python 3.7.3](https://www.python.org/downloads/) or similar version
+
+## Installation
+
```

Comment:
Wouldn't the the python script need to be run in a blank Unity project with a project name bitnaughts?

Based on line 16 in init.py
```python
d_name = os.path.dirname(abs_path + '/bitnaughts/Assets/')
```

Reply:
The instructions specify to clone this repository first which contains the bitnaughts folder so the script should run properly. However, separate Unity instructions still be worthwhile.
@Mutilar

Reply:
@GateauXD, look at the repository. The project is already created. All they have to do is clone it. Then, after running init.py, double click a scene file to open the project.